src/include/migraphx/operation.hpp

```diff
@@ -1123,6 +1205,13 @@ struct operation
                 char(0), private_detail_te_value, ctx, output, input);
         }
 
+        bool runs_on_offload_target() const override
+        {
+
```

Comment:
remove whitespace

Reply:
Same as before, it is auto-gen.

Reply:
all good as long as that's known that makes sense.